resources/test-invoker.mjs

```diff
@@ -24,7 +24,24 @@ export class TimerTestInvoker extends TestInvoker {
     }
 }
 
-export class RAFTestInvoker extends TestInvoker {
+class AsyncTimerTestInvoker extends TestInvoker {
```

Comment:
Do we plan to use both the timer and RAF invoker for async? I had a sense that we kept the sync timer invoker around for backwards compat or non-default testing environments.

I'm assuming the overall properties are quire different once we are async (potentially reducing utility of timer), and it's complex enough to reason about one implementation, so should we consider just having one and exposing it on both the default and non-default configuration?

I'm open to having both but would like to find ways to reduce the amount of work if possible :). @camillobruni @smaug---- @rniwa thoughts?

Reply:
I'd be fine to drop the setTimeout approach altogether since we haven't really discovered additional issues with it?

Reply:
Yeah, I think we should just remove the timer-based test invoker.

Reply:
Can we do the removal in a separate PR though?

Reply:
Either way is fine for me, but yeah, I don't think the timer based runner is needed in any form.